.gitignore

```diff
@@ -12,6 +12,7 @@ target
 tmp
 *.class
 *.iml
+*.run
```

Comment:
What is this ignore for?

Reply:
That is for an intellij run profile.

Reply:
let's move that into a separate hotfix commit

Reply:
Done.